### DISCUSSION TITLE
808

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#983] [Comments]
# 808

최근 독서 모임에서 다음과 같은 질문이 있었다."주인이 아닌데 어떻게 주인의식을 가질 수 있죠?"이 질문은 구직자나 팀원이 아닌, 창업을 준비하는 예비 대표님의 질문이셨다.그간의 직장 생활 경험으로 오너십을 가진 팀원은 유니콘과 같은 환상속에만 있는 것은 아닌가? 이런 사람들을 채용할 수 있을까? 보통의 팀원들에겐 어떻게 오너십을 가지게 할 수 있을까? 등에 관한 질문이셨다.이 질문에 대한 여러 사람들의 답변이 있었는데,바로 옆자리에 계셨던 태일님의 답변이 너무 좋았다."집 주인이 아닌 세입자로 살고 있어도 우린 그 집을 깨끗하게 가꾸고 꾸민다.그게 주인의식이라고 생각한다."짧은 한 문장이였지만 굉장히 크게 와닿았다.전세/월세를 살다보면 어차피 내 집도 아닌데, 그냥 대충 살고 관리 안하고 있다가 떠나면..

https://jojoldu.tistory.com/808

**Comment by mag123c** [comment]
거주 공간이 책임감이라는 말로 표현할 수 있을지 모르겠지만, 회사에서 책임감을 가지고 회사의 목표를 같이 바라보면서 갈 수 있다면 그게 긍정적인 주인의식이 되지 않을까 싶어요. 좋은 글 잘읽었습니다.
